core/src/main/java/org/jivesoftware/spark/ui/login/LoginSettingDialog.java

```diff
@@ -86,7 +86,7 @@ public boolean invoke( JFrame owner )
         }
         if ( !Default.getBoolean(Default.CERTIFICATES_MANAGER_DISABLED))
         {
-        	tabbedPane.addTab( Res.getString( "tab.certificates" ), certManager );
+        //	tabbedPane.addTab( Res.getString( "tab.certificates" ), certManager );
```

Comment:
Although this will obviously do the trick, isn't the purpose of this `if` statement to enable/disable things, from the value of `CERTIFICATES_MANAGER_DISABLED`? Shouldn't we use that?